src/naive/date.rs

```diff
@@ -252,6 +251,18 @@ impl NaiveDate {
         }
     }
 
+    /// Skips the range check on years.
+    ///
+    /// FOR INTERNAL USE ONLY.
```

Comment:
Adding this warning seems pointless since this method is already private to the module.